UNITTESTS/features/cellular/framework/AT/at_cellularcontext/unittest.cmake

```diff
@@ -9,7 +9,7 @@ set(unittest-includes ${unittest-includes}
   ../features/cellular/framework/common
   ../features/cellular/framework/AT
   ../features/cellular/framework/device
-  ../features/netsocket/cellular
+  ../features/cellular
```

Comment:
```suggestion
```
The path was wrong before (cellular wasn't in netsocket) but we never had problems building unit tests, which means it's probably not needed anyway.

Reply:
@LDong-Arm Yes, that's why I just corrected with the proper path. I am not sure whether cellular refactoring PR#13261 has done any changes related to the path if not I will remove `../features/cellular` path reference from unittest.cmake in this PR.